akka-testkit/src/main/scala/akka/testkit/TestFSMRef.scala

```diff
@@ -99,4 +99,14 @@ object TestFSMRef {
     val impl = system.asInstanceOf[ActorSystemImpl] //TODO ticket #1559
     new TestFSMRef(impl, Props(factory), impl.guardian.asInstanceOf[InternalActorRef], name)
   }
+
+  def apply[S, D, T <: Actor: ClassTag](factory: ⇒ T, supervisor: ActorRef, name: String)(implicit ev: T <:< FSM[S, D], system: ActorSystem): TestFSMRef[S, D, T] = {
+    val impl = system.asInstanceOf[ActorSystemImpl] //TODO ticket #1559
```

Comment:
The TODO was already in the code on the existing `apply`. TBH I have no idea what it refers to. I thought maybe it refers to an old ticketing system since I don't see how #1559 relates here?

I'm OK with removing all the TODOs.